apiserver/paasng/paasng/platform/engine/configurations/source_file.py

```diff
@@ -49,7 +49,7 @@ def get_procfile(self, version_info: VersionInfo) -> Dict[str, str]:
         """
 
     def get_app_desc(self, version_info: VersionInfo) -> Dict:
-        """Read app.yaml/app_desc.yaml from repository
+        """Read app_desc.yaml from repository
```

Comment:
Nit: app.yaml 这个关键字去除之后，是否还有其他代码痕迹能够看出 app.yaml 是 v1 版本的文件。看要不要留个注释说明

Reply:
已添加

```
NOTE: The platform used to support "app.yaml" using the version 1 spec, support
        has been dropped because no applications use it anymore.
```